site/content/docs/5.0/components/stepped-process.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: docs
+title: Stepped process
+description: Stepped process bar used for multiple steps forms process
+group: components
+toc: true
+---
+
+## How to use
+
+Use a `nav` element with `.stepped-process` class, containing an ordered list `<ol>`  with `.stepped-process-item` class on list items.
+
+Be careful to choose short labels when using multiple steps process, otherwise label will be cut to preserve inline layout. More information can be carried on the `title` attribute of the `.stepped-process-link`.
```

Comment:
```suggestion
Use short labels for each step, otherwise they will be cut off to preserve inline layout. More information can be carried on the `title` attribute of the `.stepped-process-link`.
```

Regarding `title`, @GaetanBt shouldn't `aria-label` be used instead, in such cases?

Reply:
In the meantime, can't we put both title & aria-label ?